### PR TITLE
Adds missing config and schema fields

### PR DESCRIPTION
**Pull request by KaiVolland**
This adds needed `spring.jpa.database-platform` see: https://stackoverflow.com/a/59623399/10342669

It also adds some missing fields to the GraphQL schema.

### CODE REVIEW

File: shogun-lib/src/main/resources/graphql/shogun.graphqls [hunk 1]
```diff
@@ -12,6 +12,8 @@ type Query {
 
 type Application {
     id: ID
+    created: String
```
Comment (ahennr) [inline shogun-lib/src/main/resources/graphql/shogun.graphqls]: `Date` is more appropriate here, isn't it?

File: shogun-lib/src/main/resources/graphql/shogun.graphqls [hunk 2]
```diff
@@ -32,9 +36,18 @@ type Layer {
...
     details: JSON
     clientConfig: JSON
 }
+
+type UserGroup {
+    id: ID
+    created: String
+    modified: String
+    name: String
+}
```
Comment (ahennr) [inline shogun-lib/src/main/resources/graphql/shogun.graphqls]: minor: plz add newline.

File: shogun-lib/src/main/resources/graphql/shogun.graphqls (hunk 1 above)
Comment (KaiVolland) [reply shogun-lib/src/main/resources/graphql/shogun.graphqls]: I had issues with configuring this as Date. And as a Date ist transformed to a String for the JSON response anyway this should be ok for now.

Comment (ahennr) [reply shogun-lib/src/main/resources/graphql/shogun.graphqls]: Alright.

**Pull request merged**